Source/Core/Core/PowerPC/JitCommon/JitBase.cpp

```diff
@@ -27,11 +27,11 @@ u32 Helper_Mask(u8 mb, u8 me)
 		);
 }
 
-void LogGeneratedX86(int size, PPCAnalyst::CodeBuffer *code_buffer, const u8 *normalEntry, JitBlock *b)
+void LogGeneratedX86(u32 size, const PPCAnalyst::CodeOp* code, const u8 *normalEntry, JitBlock *b)
```

Comment:
The only requirement for `clear()` is that `empty()` returns true after it has been called. (§ 23.2.3)
I was unable to find any guarantees about its capacity in the standard draft document (n3337 source: https://github.com/cplusplus/draft/blob/n3337/papers/n3337.pdf).